docs/src/pages/[platform]/getting-started/usage/usage.react.mdx

```diff
@@ -0,0 +1,55 @@
+import { Tabs, TabItem } from '@aws-amplify/ui-react';
+import { Example, ExampleCode } from '@/components/Example';
+
```

Comment:
FYI, we have a separate task for updating this usage page with a new guide/tutorial, so this is mostly just a port of the existing content over to a new page.